src/context/BalanceContext.tsx

```diff
@@ -8,6 +7,7 @@ import {AccountInfo} from '@polkadot/types/interfaces';
 import ModalTitle from 'presentational/ModalTitle';
 import {NetworkContext} from './NetworkContext';
 import Balances from 'presentational/Balances';
+import {useAccounts} from 'src/context/AccountsContext';
```

Comment:
no need the `src`, I guess your `vim` setup auto import like this.

Reply:
yeah, it's the auto import